src/infrastructure/config.js

```diff
@@ -24,13 +24,16 @@ export function getAppConfig() {
 
 export function setAppConfig(newConfig) {
   config.Enabled = newConfig.Enabled;
+  config.ConsoleLogging = newConfig.ConsoleLogging || config.ConsoleLogging;
+  config.CacheListRefreshFrequency =
```

Comment:
Is this related to client-side caching or server-side caching?

Reply:
This relates to the client side caching I implemented using local storage that I will remove since it adds too much complexity for (likely - I should verify this) not much performance gain.

Reply:
In case you were referring to the Console Logging, there is global setting and a extension specific setting to enable console logging. Since I rarely refer to this logging, I will likely strip this back to bare minimum. I have found whenever there is an issue to resolve, debugging is the first point of call rather than logs.